rest-server/src/config/user.js

```diff
@@ -27,7 +27,8 @@ const userPutInputSchema = Joi.object().keys({
     .min(6)
     .required(),
   admin: Joi.boolean(),
-  modify: Joi.boolean(),
+  modify: Joi.boolean()
+    .required(),
```

Comment:
Why is `modify` required while `admin` is not?

Reply:
This change is just a simple substitution of https://github.com/Microsoft/pai/pull/832/files#diff-3dd2c15619e4f4cdb338de066fddb092L45

Reply:
Found admin validation in model code, will move it to param schema